app/Http/Controllers/Flight/UpdateFlightController.php

```diff
@@ -2,15 +2,17 @@
 
 namespace App\Http\Controllers\Flight;
 
+use App\Http\Actions\UpdateFlightAction;
 use App\Http\Requests\UpsertFlightRequest;
 use App\Models\Flight;
 use Illuminate\Http\JsonResponse;
 
 class UpdateFlightController
 {
-    public function __invoke(UpsertFlightRequest $request, Flight $flight): JsonResponse
+    public function __invoke(UpsertFlightRequest $request, Flight $flight, UpdateFlightAction $action): JsonResponse
```

Comment:
in many casas you may need more than one action, that is why we tend to name the param exactly as the class ` UpdateFlightAction $updateFlightAction`

Reply:
plus it is more readable when you use it below